sdk/master_board_sdk/CMakeLists.txt

```diff
@@ -19,7 +20,11 @@ set(PROJECT_USE_CMAKE_EXPORT TRUE)
 SET(PROJECT_VERSION_COMPUTATION_METHODS ROS_PACKAGE_XML_FILE)
 
 # --- OPTIONS ----------------------------------------
-OPTION(BUILD_PYTHON_INTERFACE "Build the python binding" OFF)
+IF(APPLE)
+  SET(BUILD_PYTHON_INTERFACE OFF CACHE BOOL "Build the python binding")
+ENDIF()
+
+OPTION(BUILD_PYTHON_INTERFACE "Build the python binding" ON)
```

Comment:
shouldn't this be in the `else()` of the `if(APPLE)`?

Reply:
I do not think so. As it does not compile with APPLE I force the option to be set off.